stories/components/Pill.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Pill from '../../lib/Pill';
+import Icon from '../../lib/Icon';
+import StoryItem from '../styleguide/StoryItem';
+
+storiesOf('Components', module).add('Pill', () => (
+  <div>
+    <StoryItem title="Pill">
+      <Pill onClearClick={action('clear')}>
```

Comment:
How does this look with multiple pills? Do we need a wrapper?

Reply:
A wrapper will be needed when there's multiple pills to separate them. I didn't feel it was right to include styles to separate them inside the component.